app/util/common/common.go

```diff
@@ -1,3 +1,17 @@
 package common
 
+import "strings"
+
 const ClientVersion = "0.4.0"
+
+var LingoSuffixes = []string{".lingo", ".lingo.yaml", ".lingo.yml"}
```

Comment:
This isn't a list of suffixes, which makes the code in verify.go confusing ie `os.Stat(suffix)` looks like you're just looking for the suffix. Something like "LingoFilenames" would be clearer.